R/oc_forward.R

```diff
@@ -0,0 +1,328 @@
+#'   Argentina, "FR" for France, "NZ" for the New Zealand. Multiple countrycodes
+#'   per \code{placename} must be wrapped in a list.
+#' @param language An
+#'   \href{https://en.wikipedia.org/wiki/IETF_language_tag}{IETF language tag}
+#'   (such as "es" for Spanish or "pt-BR" for Brazilian Portuguese). OpenCage
+#'   will attempt to return results in that language. If it is not specified,
+#'   "en" (English) will be assumed by the API.
+#' @param limit The maximum number of results that should be returned. Integer
+#'   values between 1 and 100 are allowed, the default is 10 (\code{oc_forward})
+#'   or 1 (\code{oc_forward_df}), respectively.
+#' @param min_confidence An integer value between 0 and 10 indicating the
+#'   precision of the returned result as defined by it's geographical extent,
+#'   (i.e. by the extent of the result's bounding box). See the
+#'   \href{https://opencagedata.com/api#confidence}{API documentation} for
+#'   details. Only results with at least the requested confidence will be
+#'   returned.
+#' @param no_annotations A logical vector indicating whether additional
+#'   information about the result location should be returned. \code{TRUE} by
+#'   default, which means that the output will not contain annotations.
+#' @param no_dedupe A logical vector (default \code{FALSE}), when \code{TRUE}
+#'   the output will not be deduplicated.
+#' @param no_record A logical vector (default \code{FALSE}), when \code{TRUE} no
+#'   log entry of the query is created and the geocoding request is not cached
+#'   by OpenCage.
+#' @param abbrv A logical vector (default \code{FALSE}), when \code{TRUE}
+#'   addresses in the \code{formatted} field of the results are abbreviated
+#'   (e.g. "Main St." instead of "Main Street").
+#' @param add_request A logical vector (default \code{FALSE}) indicating whether
+#'   the request is returned again with the results. If the \code{return} value
+#'   is a \code{df_list}, the query text is added as a column to the results.
+#'   \code{json_list} results will contain all request parameters, including the
+#'   API key used! For \code{geojson_list} this is currently ignored by
+#'   OpenCage.
+#' @param ... Ignored.
+#'
+#' @return \code{oc_forward} returns, depending on the \code{return} parameter,
+#'   a list with either
+#'   \itemize{
+#'   \item the results as tibbles (\code{"df_list"}, the default),
+#'   \item the results as JSON lists (\code{"json_list"}),
+#'   \item the results as GeoJSON lists (\code{"geojson_list"}), or
+#'   \item the URL of the OpenCage API call for debugging purposes
+#'   (\code{"url_only"}).
+#'   }
+#'   \code{oc_forward_df} returns a tibble.
+#'
+#' @seealso \code{\link{oc_reverse}} for reverse geocoding, and the
+#'   \href{https://opencagedata.com/api}{OpenCage API documentation} for more
+#'   information about the API.
+#'
+#' @export
+#'
+#' @examples
+#' \dontrun{
+#' oc_forward(placename = "Sarzeau")
```

Comment:
since the function is vectorized what about adding examples with vectors as arguments?